File: leo2-node/src/main/java/org/deku/leo2/node/data/repositories/master/StationSectorRepository.java
package org.deku.leo2.node.data.repositories.master;

import org.deku.leo2.node.data.entities.master.StationSector;
import org.deku.leo2.node.data.entities.master.StationSectorPK;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.querydsl.QueryDslPredicateExecutor;

/**
 * Created by JT on 29.06.15.
 */
public interface StationSectorRepository extends JpaRepository<StationSector,StationSectorPK>,QueryDslPredicateExecutor {
}

File: leoz-node/src/main/java/org/deku/leoz/node/data/repository/master/StationSectorRepository.kt
package org.deku.leoz.node.data.repository.master

import org.deku.leoz.node.data.jpa.MstStationSector
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.querydsl.QuerydslPredicateExecutor

/**
 * Station sector repository
 * Created by JT on 29.06.15.
 */
interface StationSectorRepository :
        JpaRepository<MstStationSector, Long>,
        QuerydslPredicateExecutor<MstStationSector>
